OWNERS_ALIASES

```diff
@@ -24,55 +24,52 @@ aliases:
     - thejoycekung # Release Manager Associate
     - wilsonehusin # Release Manager Associate
   release-team:
-    - jeremyrickard # subproject owner / 1.20 RT Lead
-    - justaugustus # subproject owner
-    - onlydole # subproject owner / 1.19 RT Lead
-    - palnabarun # subproject owner / 1.21 RT Lead
-    - reylejano # 1.23 RT Lead
-    - savitharaghunathan # subproject owner / 1.22 RT Lead
+    - justaugustus # SIG Release Chair
+    - saschagrunert # SIG Release Chair
+    - cpanato # SIG Release Technical Lead
+    - jeremyrickard # SIG Release Technical Lead
+    - puerco # SIG Release Technical Lead
```

Comment:
Prefer "subproject owner" for the comment for Chairs/TLs.
Reasoning: Subproject ownership survives leadership changes unless the contributor opts to step down

Reply:
Does that mean I should *not* add @cpanato and @puerco? Does it mean I shouldn't remove @savitharaghunathan, @palnabarun, and @onlydole that were previously listed as "subproject owners"?

Reply:
> Does that mean I should _not_ add @cpanato and @puerco?

Let's keep all leads in there.

> Does it mean I shouldn't remove @savitharaghunathan, @palnabarun, and @onlydole that were previously listed as "subproject owners"?

Leave them in. I've been taking "Release Team lead" to also mean "subproject owner for the lifecycle of the release they've led" (which is something we should probably document, if it isn't already).

The reasoning behind this is to ensure some continuity in reviews.
The active release team lead will primarily be engrossed with operationalizing the release, while previous RT leads have the opportunity to provide longer tail reviews.

Reply:
> I'm not sure I *fully* understand the distinction between a subproject owner, a release team lead, and a SIG Release Chair/TL. But I've updated it accordingly.

@jameslaverack -- can you file an issue for this? If it's not widely understood, it's worth clarifying.